### DISCUSSION TITLE
Github flavoured `readme.md` is not getting rendered poroperly

### DISCUSSION THREAD

**Discussion by Aviksaikat** [mkdocs/mkdocs#3887] [Q&A] [answered]
Hi, whenever I'm trying to align a markdown table like this it's not getting rendered by `mkdocs` though it's fine in GitHub readme.

```md
<div align="center">

table goes here

</div>
```

It's getting rendered like this 

![image](https://github.com/user-attachments/assets/9f2af061-9a7c-40eb-a401-1ab27322e1c5)

In github it's working fine

![image](https://github.com/user-attachments/assets/4148bbe6-3799-4e71-9c11-c91fb660a929)

**Comment by pawamoy** [comment, marked as answer]
MkDocs uses Python-Markdown, which doesn't follow GitHub's flavored Markdown.

To convert Markdown within HTML tags, enable the `md_in_html` extension, and set the `markdown` HTML attribute on your div:

```yaml
# mkdocs.yml
markdown_extensions:
- md_in_html
```

```html
<div align="center" markdown>

table goes here

</div>
```
